### PR TITLE
Move builders to traits

### PR DESCRIPTION
**Pull request by xduh**

### CODE REVIEW

File: lib/Customer/Phone.php [hunk 1]
```diff
@@ -4,31 +4,29 @@
...
      */
     private $number;
 
     /**
-     * @var int $ddi | DDI (Discagem Direta Internacional)
+     * @var int $ddi
      */
     private $ddi;
 
     /*
```
Comment (devdrops) [inline lib/Customer/Phone.php]: Missing extra `*` at the docblock opening.

File: lib/Payable/PayableBuilder.php [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace PagarMe\Sdk\Payable;
+
+trait PayableBuilder
+{
+   /**
```
Comment (devdrops) [inline lib/Payable/PayableBuilder.php]: Docblock identation.

File: tests/unit/AntifraudAnalyses/AntifraudAnalysisBuilderTest.php [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+     * @test
+     */
+    public function mustCreateAntifraudAnalysisCorrectly()
+    {
+        $payload = '{"object":"antifraud_analysis","name":"development","score":null,"cost":0,"status":"failed","date_created":"2017-01-03T21:03:55.922Z","date_updated":"2017-01-03T21:03:55.945Z","id":155174}';
+
+        $antifraudAnalysis = $this->buildAntifraudAnalysis(
+            json_decode($payload)
+        );
+
```
Comment (devdrops) [inline tests/unit/AntifraudAnalyses/AntifraudAnalysisBuilderTest.php]: Extra line.

**Comment by xduh** [comment]
Done

**Pull request merged**